OOps/aops.c

```diff
@@ -2219,9 +2219,7 @@ int32_t hw_channels(CSOUND *csound, ASSIGN *p){
 int32_t inRange_i(CSOUND *csound, INRANGE *p)
 {
   p->narg = p->INOCOUNT-1;
-  OPARMS oparms;
-  csound->GetOParms(csound, &oparms);
-  if (UNLIKELY(!oparms.sfread))
+  if (UNLIKELY(!csound->oparms->sfread))
```

Comment:
This is reading oparms directly while elsewhere the code was updated to use csound->GetOParms(csound). Any reason for this here?

Reply:
good catch, fixed for consistency.